PREP_SYSTEM_FOR_DIETPI.sh

```diff
@@ -1595,17 +1592,23 @@ _EOF_
 		G_RUN_CMD systemctl stop dietpi-ramdisk
 
 		# - Clear tmp files on disk
-		rm /var/tmp/dietpi/logs/* &> /dev/null
+		rm -R /var/tmp/dietpi/logs/*
 
 		# - Clear items that may have been left on disk, from previous PREP's
-		rm -R /DietPi/* &> /dev/null
+		umount /DietPi
+		rm -R /DietPi
+		mkdir /DietPi
+
 		cd /root
-		umount /tmp; rm -R /tmp/* &> /dev/null
+		umount /tmp
+		rm -R /tmp
+		mkdir /tmp
 
 		sync
 
 		# - Remove PREP script
-		rm /root/PREP_SYSTEM_FOR_DIETPI.sh &> /dev/null
+		[[ -f /root/PREP_SYSTEM_FOR_DIETPI.sh ]] && rm /root/PREP_SYSTEM_FOR_DIETPI.sh
```

Comment:
Might be easier to simply?:
```
rm $(find / -name 'PREP_SYSTEM_FOR_DIETPI.sh') &> /dev/null
```

Reply:
@Fourdee 
I just though that find might take a long time. On my RPi at least it does, of course depending on SDcard as well. And, if I remember right, it does search all `/mnt/*` drives as well, which increases the time much more. For our image creation not a big problem, if we have no external drives attached, not much data on disk etc., but for end users with perhaps already attached drives and custom data on disk, it properly leads to an unnecessary delay?